package.json

```diff
@@ -39,7 +39,7 @@
   },
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "clean:all": "lerna clean && node scripts/clean-modules/index.js",
+    "clean:all": "lerna clean --yes && node scripts/clean-modules/index.js",
```

Comment:
Thank you! I think every time I run this I move to something else to give it time to only come back to the terminal with this lerna step pending.

Reply:
You're not alone.

Reply:
We should do this for all monorepos.